test/cmdlineTests.sh

```diff
@@ -358,16 +354,16 @@ function test_via_ir_equivalence()
 
     for yul_file in $(find . -name "${output_file_prefix}*.yul" | sort -V); do
         bin_output_two_stage+=$(
-        msg_on_error --no-stderr "$SOLC" --strict-assembly --bin "${optimizer_flags[@]}" "$yul_file" |
+            msg_on_error --no-stderr "$SOLC" --strict-assembly --bin "${optimizer_flags[@]}" "$yul_file" |
```

Comment:
Are the newly added indents here (and the `sed`s below a mistake?

Reply:
No, why do you think they're a mistake?